### ISSUE TITLE
CFE declaration order can cause as3 asynchronous 503 error 

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-bigip-runtime-init#42]
Trying to deploy DO, CFE, and AS3. In relation to existing ticket https://github.com/F5Networks/f5-bigip-runtime-init/issues/41, I was able to fix the "filtered bucket" error by using the suggested order of DO (without sync), CFE, AS3, then DO (with sync stuff). My issue is that there is an asynchronous task error on appsvc most likely due to DO or the previous CFE declaration POSTing.

error...
```
Wed, 06 Apr 2022 22:19:13 GMT - warning: [appsvcs] {"message":"Unable to send telemetry data: 503 {\"code\":503,\"message\":\"There is an active asynchronous task executing.\",\"errorStack\":[],\"apiError\":32964609}","level":"warning"}
```

When the cluster tries to failover, you are advised to reset CFE state.
```
Wed, 06 Apr 2022 22:21:50 GMT - severe: [f5-cloud-failover] Recovery operations are empty, advise reset via the API Error: Recovery operations are empty, advise reset via the API
    at FailoverClient._getRecoveryOperations (/var/config/rest/iapps/f5-cloud-failover/nodejs/failover.js:469:19)
    at _getDeviceObjects.then.then (/var/config/rest/iapps/f5-cloud-failover/nodejs/failover.js:130:33)
    at tryCatcher (/usr/share/rest/node/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/usr/share/rest/node/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/usr/share/rest/node/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/usr/share/rest/node/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
```

## Troubleshooting
Tried to increase restjavad memory and also change order of operations.

Also uploaded qkview with -s0 to pre-sales internal ticket.

**Comment by JeffGiroux** [comment]
I had to make sure my bucket was empty. I redeployed and no longer have the async 503 error in appsvc depending on how I do the declaration order.

If I do DO (no sync) > CFE > AS3 > DO (with sync), then I tested once and did not receive 503 appsvc async on either node. Once I tested my original config of DO (sync) > CFE > AS3, then node 1 ran into the 503 appsvc async error whereas node 2 had no errors. As a result, node 2 pushed AS3 and synced cluster so node 1 also had the as3 app. 

I would need to test more, but order of operations is important as well as an EMPTY bucket.
